changelog.txt

```diff
@@ -1,3 +1,25 @@
+13.1 (2023-04-10)
+------------------------------------------------------------------------
+Feature: [NewGRF] Engine name callback. (#10399)
+Change: Avoid crashing to the side of a train (#10496)
+Fix 8361cf5a73: Missing bounds check for house specs. (#10625)
+Fix: Update top toolbar tooltips for added features. (#10616)
+Fix: NewGRF Action3 (+others) validation (#10601)
+Fix: Clicking on editbox clear button didn't take account of padding. (#10583)
+Fix: [Script] Access to enum/consts defined outside of main.nut (#10573)
+Fix #10568: "can savegame be loaded check" failed in dedicated server (#10572)
+Fix #10554: Let Scrollbar::SetPosition clamp instead of assert. (#10555)
```

Comment:
Fix #10554: Crash when scrolling in the autoreplace window with collapsed variants (#10555)